src/main/java/org/aion/wallet/account/AccountManager.java

```diff
@@ -320,13 +299,28 @@ private String getStoredAccountName(final String publicAddress) {
         return walletStorage.getAccountName(publicAddress);
     }
 
+    private AccountDTO getNewAccount(final String publicAddress, boolean isImported, int derivation) {
+        return new AccountDTO(getStoredAccountName(publicAddress),
+                publicAddress,
+                getFormattedBalance(publicAddress),
+                currencySupplier.get(),
+                isImported,
+                derivation,
+                new TreeSet<>(transactionComparator));
```

Comment:
`transactionComparator` can be moved as a `private static final` in `AccountDTO`

Reply:
Also, the `TransactionComparator` class could be moved as inner to `AccountDTO`